staging/src/k8s.io/controller-manager/pkg/features/kube_features.go

```diff
@@ -21,23 +21,11 @@ import (
 )
 
 const (
-	// Every feature gate should add method here following this template:
```

Comment:
This should not be excluded, as it's an example for people willing to change this file